coordinator/internal/logic/provertask/batch_prover_task.go

```diff
@@ -271,17 +288,13 @@ func (bp *BatchProverTask) getBatchTaskDetail(dbBatch *orm.Batch, chunkInfos []*
 	}
 	taskDetail.BatchHeader = batchHeader
 	taskDetail.BlobBytes = dbBatch.BlobBytes
-
-	if len(dbBatch.BlobDataProof) < 160 {
-		return nil, fmt.Errorf("blob data proof length is less than 160 bytes = %d, taskID: %s: %s", len(dbBatch.BlobDataProof), dbBatch.Hash, common.Bytes2Hex(dbBatch.BlobDataProof))
-	}
-
+	taskDetail.ChallengeDigest = common.HexToHash(dbBatch.ChallengeDigest)
```

Comment:
There is no clue for new developers that where these fields come from (in fact they are generated inside of `da-codec`), maybe we should add a comment about that